game/src/game/components/VehicleComponent.h

```diff
@@ -31,11 +32,13 @@ class VehicleComponent final : public Component,
     jss::object_ptr<Transform> transform_;
     jss::object_ptr<PhysicsService> physics_service_;
     jss::object_ptr<InputService> input_service_;
+    jss::object_ptr<GameStateService> game_state_service_;
 
     /* variables for vehicle */
 
     // The vehicle with direct drivetrain
     DirectDriveVehicle g_vehicle_;
+    std::string g_vehicle_data_path_ = "resources/vehicle_data/";
```

Comment:
Can we remove this? I dont think this path should change between vehicles, and it's set as a constant in the cpp file